html/changepage.html

```diff
@@ -24,42 +24,57 @@ <h1 class="header__page-title">
+                                </path>
+                            </svg>                               
                             <span> Home</span>
                         </a>
                     </li>
                     <li class="side-nav__item">
                         <a href="./project-page.html" class="side-nav__link">
-                            <svg class="side-nav__icon">
-                                <use xlink:href="img/sprite.svg#icon-aircraft-take-off"></use>
+                            <svg class="side-nav__icon" version="1.1" xmlns="http://www.w3.org/2000/svg" width="32"
+                                height="32" viewBox="0 0 32 32">
+                                <path
+                                     d="M32 10l-16-8-16 8 16 8 16-8zM16 4.655l10.689 5.345-10.689 5.345-10.689-5.345 10.689-5.345zM28.795 14.398l3.205 1.602-16 8-16-8 3.205-1.602 12.795 6.398zM28.795 20.398l3.205 1.602-16 8-16-8 3.205-1.602 12.795 6.398z">
+                                </path>
                             </svg>
                             <span>Project</span>
                         </a>
                     </li>
                     <li class="side-nav__item side-nav__item--active">
                         <a href="./changepage.html" class="side-nav__link side-nav__link--active">
-                            <svg class="side-nav__icon">
-                                <use xlink:href="img/sprite.svg#icon-key"></use>
+                            <svg class="side-nav__icon" version="1.1" xmlns="http://www.w3.org/2000/svg" width="24"
+                                height="24" viewBox="0 0 24 24">
+                                <path
+                                    d="M17.672 9.516l-7.078 7.078-4.266-4.219 1.453-1.406 2.813 2.813 5.672-5.672zM3.984 12q0-1.734 0.727-3.258t1.898-2.602l2.391 2.344v-6h-6l2.203 2.203q-1.453 1.359-2.32 3.258t-0.867 4.055q0 1.969 0.68 3.68t1.898 3.047 2.859 2.18 3.563 1.031v-2.016q-1.5-0.188-2.766-0.867t-2.227-1.758-1.5-2.414-0.539-2.883zM21.984 12q0-1.969-0.68-3.68t-1.898-3.047-2.859-2.18-3.563-1.031v2.016q1.5 0.188 2.766 0.867t2.227 1.758 1.5 2.414 0.539 2.883q0 1.734-0.727 3.258t-1.898 2.602l-2.391-2.344v6h6l-2.203-2.203q1.453-1.359 2.32-3.258t0.867-4.055z">
+                                </path>
                             </svg>
                             <span>Changes</span>
                         </a>
                     </li>
                     <li class="side-nav__item">
-                        <a href="#" class="side-nav__link">
-                            <svg class="side-nav__icon">
-                                <use xlink:href="img/sprite.svg#icon-map"></use>
+                        <a href="../html/branch.html" class="side-nav__link">
```

Comment:
you can change the ../html/branch.html to just ./branch because its in the same folder with your branch

Reply:
you can do this to all the branches except main/index.html